Yoast/yoast-components
# Polish the loading-spinner icon.

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

Polishes the loading-spinner:
- builds the CSS necessary to see the icon spinning in the standalone app
- removes an unused animation (was transitioning color through the same color)
- also, lists the icon in alphabetical order

## Test instructions

before the patch:
- go to the standalone app (`yarn start` and go to `localhost:3333`)
- go to the SVG icons tab
- see the loading-spinner doesn't show anything

after the patch:
- go to the standalone app 
- go to the SVG icons tab
- refresh the page if necessary]
- see the loading-spinner is now spinning
- check it looks as expected (note: in Firefox you will see something a bit weird, that's just how the icon works)
- see icons are in alphabetical order

Note: the standalone app is meant for testing thus we should ensure components can be effectively tested. Having an icon that can't be visually tested is a bit pointless.

Fixes #719
